analyzers/Merger.py

```diff
@@ -0,0 +1,19 @@
+from heppy.framework.analyzer import Analyzer
+import copy
+
+
+
+class Merger(Analyzer):
```

Comment:
Hi Colin
Thanks for the comments which I have addressed and pushed.

I am not 100% sure what “add doctest” means. I have added some documentation to the Merger class and hope this is what you meant. If  you meant something further just let me know!

Alice



> On 12 May 2016, at 13:50, Colin Bernet <notifications@github.com> wrote:
> 
> In analyzers/Merger.py <https://github.com/cbernet/heppy/pull/11#discussion_r63006333>:
> 
> > @@ -0,0 +1,19 @@
> > +from heppy.framework.analyzer import Analyzer
> > +import copy
> > +
> > +
> > +
> > +class Merger(Analyzer):
> Hi Alice,
> 
> Thanks for these modifications!
> Could you please add doctest to this class as for other analyzers?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/cbernet/heppy/pull/11/files/9ef45c280d4716b40c8a0a0baa6fa70135de3ea8..23a1957e5a58bb348cb15e49ba4d92b38950031e#r63006333>